.github/workflows/daily.yaml

```diff
@@ -129,7 +129,10 @@ jobs:
           chmod +x ./quarkus-dev-cli
       - name: Test in Native mode
         run: |
-          mvn -fae -V -B -s .github/mvn-settings.xml -fae clean verify -Dnative \
+          # Skip module due to high time execution on Native
```

Comment:
Is long execution time trouble for daily run?

Reply:
Yes... I'm planning to run a parallel execution for this module.